README.md

```diff
@@ -1,17 +1,93 @@
 [![Build Status](https://travis-ci.org/Nexapp/NexappCore.svg?branch=master)](https://travis-ci.org/Nexapp/NexappCore)
 
 # Nexapp Core
-Several classes that we use everyday.
+Several Java mini-modules for REST APIs.
 
 ## Provided by the library
-**ServiceLocator**: The super lightweight anti-pattern to inject dependencies easily. Generally combined with the BaseContext class. <br />
-**EmailValidator**: Perform email validations. <br />
-**NumberPresenter**: A helper to present numbers in various forms (rounded or not, with n decimals, etc.) <br />
-**DateProvider**: An interface to provide date and timestamp. Mostly use for mocking. There is a Java implementation as well. <br />
-**Range**: Can handle two comparables (overlapping for example). <br />
-**Pagination**: Structure that represents the pagination concept (page, itemPerPage, startingIndex). <br />
-**Picker**: Pick x item(s) randomly from a Collection. <br />
-**WeightedSet**: Pick a random item from a Collection. Each item appear x times according to their weight in the Set. <br />
+**DateProvider**
+```java
+// Useful for mocking purposes
+DateProvider dateProvider = new JavaDateProvider();
+
+// Without timezone
```

Comment:
I'd specify these as "legacy" and shouldn't be used

Reply:
I'm unsure if this is really legacy. In fact, they are usable but not recommended. Java expose them as is, but we [Nexapp] prefer use the zoned version.